### PR TITLE
stabilize path operations, optimize remove_comments, add initial sample for documentation

### PR DESCRIPTION
**Pull request by test-fullautomation**

### CODE REVIEW

**Comment by test-fullautomation** [comment]
Hi Son,
can you please review and test this changes with testsuites-management?
* I stabilized path functions. It was e.g. not possible to use relative paths when used by a python module (I created sample.py for this).
* I optimized the code for "remove_comments". Also /* */ multiline-comments are working now.
* I added a start for python samples.
Thank you,
 Thomas

**Comment by namsonx** [comment]
Hello Thomas,

     I will test with testsuites-management package and inform you the result.

Thank you,
Son

**Comment by test-fullautomation** [comment]
Hi Son,
thank you, but we need to talk before.
I assume that it will be not compatible to testsuites-management now. The atest is also not working anymore.
I have to explain why and we need to find the further strategy.
Thank you,
 Thomas

File: sample/sample.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import sys
+sys.path.append('D:\B\python-jsonpreprocessor')
+
+import JsonPreprocessor
+from pprint import pprint
+
+prepro=JsonPreprocessor.CJsonPreprocessor()
+
+data=prepro.jsonLoad(".\json\json_with_comment.json")
```
Comment (test-fullautomation) [inline sample/sample.py]: Hi Son,
important is that this use-case is working.
We assume that most of the users will not require this for robotframework, but for any  other python project.
Usually in this case the configuration data is either relative to the project, or refrerenced by means of an environment variable.
Thank you,
 Thomas

File: JsonPreprocessor/CJsonPreprocessor.py [hunk 2]
```diff
@@ -169,41 +223,22 @@ def __processImportFiles(self, input_data):
...
+            if s.startswith('/'):
+                return "" 
+            else:
+                return s
+      
+        file=open(jsonFile,mode='r')
+        sContent=file.read()
+        file.close()
+
+        pattern = re.compile(r'//.*?$|/\*.*?\*/|\'(?:\\.|[^\\\'])*\'|"(?:\\.|[^\\"])*"', re.DOTALL | re.MULTILINE)
```
Comment (test-fullautomation) [inline JsonPreprocessor/CJsonPreprocessor.py:306]: This allows /* */ multiline comments, too.

File: JsonPreprocessor/CJsonPreprocessor.py [hunk 3]
```diff
@@ -134,6 +136,50 @@ def __init__(self, syntax=CSyntaxType.json, currentCfg={}):
         self.currentCfg = currentCfg
         self.lUpdatedParams = []
             
+    #
```
Comment (test-fullautomation) [inline JsonPreprocessor/CJsonPreprocessor.py]: Stable path functions taken from TML Framework.

**Comment by test-fullautomation** [comment]
Must be tested and then merged together with
https://github.com/test-fullautomation/robotframework-testsuitesmanagement/pull/19

**Pull request merged**